clang/lib/AST/ByteCode/Compiler.h

```diff
@@ -52,12 +52,14 @@ struct InitLink {
     K_Decl = 3,
     K_Elem = 5,
     K_RVO = 6,
-    K_InitList = 7
+    K_InitList = 7,
+    K_DIE = 8,
```

Comment:
What does `DIE` stand for here? The rest of the fields are mostly self explanatory. Either a different name or a comment would help. Also `DIE` is a dwarf think and so we are now overloading the term, which is not good.